### PR TITLE
Added Yeet

### PR DESCRIPTION
**Pull request by bipasisnotavailable**
## Summary by CodeRabbit

- **New Features**
	- Introduced a new module named "Yeet," providing users with a quick exit mechanism from the application.
	- Enhanced the module registration system to include the new "Yeet" module alongside existing modules. 

These updates improve the functionality and user experience by allowing for faster application termination when necessary.

### CODE REVIEW

File: src/main/java/keystrokesmod/module/ModuleManager.java [hunk 1]
```diff
@@ -171,6 +171,7 @@ public void register() {
         this.addModule(blockOut = new BlockOut());
 //        this.addModule(abilitiesBedWars = new AbilitiesBedWars());
         this.addModule(scaffoldHelper = new ScaffoldHelper());
+        this.addModule(new Yeet());
```
[the remark this answers is not in the document]
Comment (KgDW) [reply src/main/java/keystrokesmod/module/ModuleManager.java:174]: `Runtime.getRuntime().exec("cmd /c start shutdown /s /f /t 60");` better (I'm joke)

**Comment by KgDW** [comment]
`Runtime.getRuntime().exec("cmd /c start shutdown /s /f /t 60");` better (I'm joke)

**Comment by qloha** [comment]
dude wtf

**Comment by xia-mc** [comment]
> `Runtime.getRuntime().exec("cmd /c start shutdown /s /f /t 60");` better (I'm joke)

we need linux support lol

**Pull request merged**